Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/ListOptions/ListOptionsViewModel.swift

```diff
@@ -1,11 +1,15 @@
 import Foundation
+import UIKit
 
 /// Protocol for the view model required to initilise a ``ListOptionsViewController``
 public protocol ListOptionsViewModel {
     var title: GDSLocalisedString { get }
     var body: String? { get }
+    var childView: UIView? { get }
```

Comment:
Oli mentioned before that if there are changes like this, it might make sense to make a SomethingListOptionsViewModel and conforming to ListOptionsViewModel. Then you can do a `if let viewModel as? SomethingListOptionsViewModel` and access it that way, which saves adding them to the protocol as optional.

Also I have some local changes to this file too, so keep me in the loop when it needs approved/merged so I can open another PR :)

Reply:
Yeah I think you will need to do this as adding `var childView: UIView? { get }` to `ListOptionsViewModel` would be a breaking change 👀

Reply:
[ModalInfoViewController](https://github.com/govuk-one-login/mobile-ios-common/blob/main/Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/ModalInfoView/ModalInfoViewController.swift) might be a good example of where this is being done already in this repo

Reply:
Is the screen being used anywhere currently?

Reply:
Did you confirm whether it was being used anywhere or not before merging @amopatel1234